<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemaWidget</class>
 <widget class="QWidget" name="SchemaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="fill_color_tb">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>5</y>
     <width>45</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>23</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="show_rect_chk">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>6</y>
     <width>123</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Show rectangle</string>
   </property>
  </widget>
  <widget class="QLabel" name="fill_color_lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>4</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Fill color:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
